src/Illuminate/Contracts/Database/Eloquent/HasCasterClass.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Illuminate\Contracts\Database\Eloquent;
```

Comment:
Does this really belong in the contracts package?

Reply:
I followed what `\Illuminate\Contracts\Database\Eloquent\CastsAttributes` and `\Illuminate\Contracts\Database\Eloquent\CastsInboundAttributes` did.